github/misc.go

```diff
@@ -107,19 +107,19 @@ func (c *Client) ListCodesOfConduct(ctx context.Context) ([]*Conduct, *Response,
 	// TODO: remove custom Accept header when this API fully launches.
 	req.Header.Set("Accept", mediaTypeCodesOfConductPreview)
 
-	var conducts []*Conduct
-	resp, err := c.Do(ctx, req, &conducts)
+	var codeOfConducts []*CodeOfConduct
+	resp, err := c.Do(ctx, req, &codeOfConducts)
 	if err != nil {
 		return nil, resp, err
 	}
 
-	return conducts, resp, nil
+	return codeOfConducts, resp, nil
 }
 
-// ListCodeOfConductByKey returns an individual code of conduct.
+// GetCodeOfConductByKey returns an individual code of conduct.
 //
 // https://developer.github.com/v3/codes_of_conduct/
```

Comment:
You can make the URL more specific:

https://developer.github.com/v3/codes_of_conduct/#get-an-individual-code-of-conduct